### DISCUSSION TITLE
XPDR Button on Stream Deck Plus

### DISCUSSION THREAD

**Discussion by Panthros** [nguyenquyhy/Flight-Tracker-StreamDeck#401] [General]
I just bought the Stream Deck Plus and love Flight-Tracker, thank you.  I am using 0.50.  I have Com1/Com2 and Nav1/Nav2 working.  You hold a button and then use the outer and inner buttons to change frequency.  I have added the XPDR and it displays but when I hold the XPDR button nothing happens.  How do I change the XPDR?  What would be perfect is use the outer button to change the first two numbers and the inner button changes the last two numbers.

**Comment by vcsandstorm** [comment]
This is one of the better plugins for Stream deck plus. However, I have the same question for the XPDR. And more pressingly will be a customize dial.
